### ISSUE TITLE
Section-Notice: issues with CTA only variant

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#2146]
CTA should be positioned under body on small screen in all cases, even when there is no dismiss button:
<img width="387" alt="Screenshot 2023-09-01 at 11 30 19 AM" src="https://github.com/eBay/skin/assets/38065/7501eb44-90c9-4a42-93c6-217b5cece7ee">

We can see that when there is a CTA with dismiss button, the CTA is positioned correctly.

<img width="489" alt="Screenshot 2023-09-01 at 11 26 41 AM" src="https://github.com/eBay/skin/assets/38065/999db58c-59d3-44c5-8484-ecd15fb9a7f5">

When we replace the dismiss button with the CTA, the CTA loses the ability to go in the correct space under the body:
<img width="492" alt="Screenshot 2023-09-01 at 11 26 30 AM" src="https://github.com/eBay/skin/assets/38065/df41b2a7-0fcd-46e7-b9c1-511da465d151">

This behavior can be verified in current storybook: https://opensource.ebay.com/skin/storybook/?path=/story/skin-section-notice--information-with-button

**Comment by agliga** [comment]
We should rework notice, this should be done for all notices. We should rework cta/footer as well as in ebayui.

**Comment by ArtBlue** [comment]
Is this really a bug? Originally, the placement of various items was related to whether other items were present. I recall this being the desired effect by DS. We need to verify.

**Comment by agliga** [comment]
Yes we have a lot of different ways to show things in notice that we tacked on. It needs a refactor
